extra/1-radio-stations.js

```diff
@@ -14,17 +14,56 @@
  */
 
 // `getAllFrequencies` goes here
+function getAllFrequencies() {
```

Comment:
Hey man so on this **getAllFrequencies function** I am sure typing all those number's your self was a tedious task.

- Below is the logic behind programmatically getting an array from 80-108

### Thought Process To Apply
1.  First thing you have to know between this two bounds **80 & 108 inclusive** how many numbers are there.
2. Now that you know how many numbers are the between the two bounds you can set a counter that will help you with **proper indexing and stopping at the upper bound** 
3. However obviously you want to be inserting numbers from lowerbound to upperbound i.e 80-108 so you can have a start value ```let startValue = 80;``` increment that on each iteration then you should programmatically fill the array with 80-108

## Hope this helps 🔥💪